pyproject.toml

```diff
@@ -1,21 +1,21 @@
-[tool.poetry]
+[project]
 name = "tom-demo"
 version = "1.0.7"
```

Comment:
I'm a fan of dynamic versioning. (see `tom_swift` for example), but don't worry about it if it doesn't sound fun.